.ipynb_checkpoints/README-checkpoint.md

```diff
@@ -0,0 +1,20 @@
+
+Analogous to the human brain, a neural network involves ‘neurons’ that can communicate with each other. These neurons are organized in several layers. The first layer is the input layer, the next are the ‘hidden layers’, followed by the output layer. Each neuron in a layer is actually a number. This number is called the ‘activation’ of that neuron. It determines the extent to which the pattern (e.g., visual, sound, etc.) picked up by that neuron weighs in on the output of that layer. 
+
+The activations in one layer are propagated forward to the next layer as a weighted sum. That is, the input of a neuron in one layer is the weighted sum of the outputs (i.e., activations) of the neurons in the previous layer. In addition, each neuron has a bias. The bias of a neuron can be thought of as its ‘threshold’ – its tendency to be on or off. This bias is similar to the intercept in a regression equation. In effect, the activations of the neurons in the preceding layer, the weights and the biases combine to determine the activation of a neuron. This activation value goes into an activation function, which converts it into the output of that neuron. I will use the sigmoid function for simplicity, which compresses the activation value into a number between zero and one. 
+
+The network reaches the correct response by learning from its mistakes. The quality of a response is communicated back to the network via back propagation. A loss function is defined, such as the sum of the squared differences. This calculates the distance between the correct response and the predicted response. The derivative of the loss function with respect to the weights gives the change in the loss function based on the change in the weights. The aim is to find the weights and the biases that minimize the loss function. This is done by an algorithm called the gradient descent. The gradient of a function is the direction in which it increases the most steeply. The negative of the gradient gives us the steepest decrease. The neural network minimizes the loss function by iterating according to the gradient descent. One can picture sliding on the slope of a function until we reach the lowest point on the curve. 
+
+To summarize, these are the functions I will use in this project. 
+1. The activation function defines the nodes and their outputs
+2. The weights and the biases determine the inputs to subsequent layers 
+3. The loss function is the feedback mechanism
+4. Gradient descent optimizes learning from the feedback  
```

Comment:
Clear but I agree, could do with some elaboration especially regarding the time frame you might be looking at to complete it all.